sentry/sentry.py

```diff
@@ -46,22 +54,57 @@ def cog_unload(self):
         self.bot.remove_before_invoke_hook(self.before_invoke)
         return super().cog_unload()
 
+    @checks.mod()
+    @commands.group(name="sentry", pass_context=True)
+    async def _sentry(self, ctx: commands.context.Context):
+        """Automatically reply to messages matching certain trigger phrases"""
+
+    @_sentry.command(name="set_env")
+    async def sentry_set_env(self, context: commands.context.Context, new_value: str):
+        """Set sentry environment"""
+        if not context.guild:
+            await context.send("That command is not available in DMs. Sentry environment is a per-guild setting.")
+            return
+        await self.config.guild(context.guild).environment.set(new_value)
```

Comment:
It looks like this doesn't ensure the value is unique across guilds. Is this intended behaviour?

Reply:
Yeah that's on purpose, multiple guilds can send to the same environment without issues.